### PR TITLE
CI: create separate environment for testing

### PR DESCRIPTION
**Pull request by glemaitre**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -1,23 +0,0 @@
-name: ramp
-dependencies:
-  - numpy
-  - pyyaml
```
Comment (jorisvandenbossche) [inline environment.yml]: Why do you delete this one?

Comment (jorisvandenbossche) [reply environment.yml]: Ah, or you only renamed it to `test_environment.yml` ? (for me, just `environment.yml` is fine for the base environment to set up the full server/frontend/database/worker thing)

Comment (glemaitre) [reply environment.yml]: I renamed it test_environment.yml. I can revert the renaming

File: .travis.yml [hunk 2]
```diff
@@ -13,9 +13,9 @@ before_install:
     - export PATH=/home/travis/miniconda/bin:$PATH
     - conda update --yes --quiet conda
 install:
-    - conda create -n ramp --yes pip python=$PYTHON_VERSION ipykernel
-    - source activate ramp
-    - conda env update --file environment.yml
+    - conda env create -f test_environment.yml python=$PYTHON_VERSION
+    - conda env create -f test_environment_iris_kit.yml python=$PYTHON_VERSION
+    - source activate testenv
```
Comment (jorisvandenbossche) [inline .travis.yml]: Where is the name 'testenv' coming from?

File: environment.yml (hunk 1 above)
Comment (jorisvandenbossche) [reply environment.yml]: BTW, the `test_environment.yml` file seems empty

File: .travis.yml (hunk 2 above)
Comment (glemaitre) [reply .travis.yml]: normally this is the environment created by `environment.yml`. It seems that it named it `python=3.7` instead. I have to check why.

File: environment.yml (hunk 1 above)
Comment (glemaitre) [reply environment.yml]: Uhm. What do you mean?

**Comment by glemaitre** [comment]
The CIs are happy (at least for the engine).

**Comment by glemaitre** [comment]
@jorisvandenbossche I think that it can be merged

File: environment.yml (hunk 1 above)
Comment (jorisvandenbossche) [reply environment.yml]: it just looked empty in the diff, but now you renamed it again, the diff looks good

**Comment by jorisvandenbossche** [comment]
python 2 is failing, that is a known problem?

**Comment by glemaitre** [comment]
Yes this is due to the ramp board.

**Comment by glemaitre** [comment]
Then merging

**Pull request merged**